Tests/Foundation/Tests/TestOperationQueue.swift

```diff
@@ -13,7 +13,7 @@ class TestOperationQueue : XCTestCase {
     static var allTests: [(String, (TestOperationQueue) -> () throws -> Void)] {
         return [
             ("test_OperationPriorities", test_OperationPriorities),
-            /* ⚠️ */ ("test_OperationCount", testExpectedToFail(test_OperationCount, "OperationQueue can't count to more than 1")),
+            ("test_OperationCount", test_OperationCount),
```

Comment:
Might be nice to have a test that ensures that the operations returned are also correct.  I don't think that we need to hold up this particular patch for that, but would be a good addition to the test suite in a follow up change.